install/kubernetes/cilium/templates/clustermesh-apiserver/service.yaml

```diff
@@ -35,4 +35,8 @@ spec:
   {{- if .Values.clustermesh.apiserver.service.internalTrafficPolicy }}
   internalTrafficPolicy: {{ .Values.clustermesh.apiserver.service.internalTrafficPolicy }}
   {{- end }}
+  sessionAffinity: ClientIP
+  sessionAffinityConfig:
+    clientIP:
+      timeoutSeconds: 600
```

Comment:
non-blocking question: is there any reason why we set it to 600?
I guess a lower value could help with testing in CI, but probably 10 minutes is still too high. 
I would probably lower it to 1 minute for testing and leave the default 3h for production.

Reply:
Yeah it seems like the higher the better, since on failure a client will be connected to another backend and we can detect and correct that right away. I've removed the sessionAffinityConfig to use the default 3h